app-privacy-policy.md

```diff
@@ -76,24 +74,30 @@ Logged HTTP requests are stored for up to 3 months.
 ### Location
 
 The server list is currently located in the Netherlands.
-By using this service, you give permission for the data to be moved as needed.
 
 
-## Browsing Online Content (ContentDB)
+## ContentDB
 
 You can find more information on
 [ContentDB's privacy policy](https://content.minetest.net/privacy_policy/).
 
 ### Information collected
 
-Requests to [ContentDB](https://content.minetest.net) will be made when using
-the ContentDB dialog found at Content > Browse online content.
-The following information will be transferred or included:
+Requests to [ContentDB](https://content.minetest.net) will be made in the main
+menu when using the ContentDB feature. This includes checking for updates then
+you have packages from ContentDB installed. The following information will be
+transferred or included:
 
 * IP address
 * Page URL
 * Minetest version
-* Platform, OS, device, etc
+* Platform and Operating System
+
+for example:
+
+```
+11.22.33.44 content.minetest.net - [06/July/2024:10:05:00 +0200] "GET /packages/Wuzzy/glitch/releases/18414/download/?reason=new HTTP/2.0" 302 233 "-" "Minetest/5.8.0 (Windows/10.0.19041 x86_64)"
+```
```

Comment:
should mention the above is for _all_ requests and there is some data sent by other requests (like which packages you view/install, and why). This is covered in the below section but should be listed here too